### DISCUSSION TITLE
HTML // please help me

### DISCUSSION THREAD

**Discussion by baranauxyz** [community/community#21475] [Other Feature Feedback, Questions, & Ideas]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/b54d7988a8771b1003c5bd562fde38ac64d9a130.png" data-download-href="/uploads/short-url/pRSptXqtIEWxrDUNuRoZqRnu2SQ.png?dl=1" title="7q-3qBwkteA" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/57573260/181064912-0efaa82a-14b7-4c26-9efe-8950a91a1bf7.png" alt="7q-3qBwkteA" data-base62-sha1="pRSptXqtIEWxrDUNuRoZqRnu2SQ" width="690" height="306" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/b54d7988a8771b1003c5bd562fde38ac64d9a130_2_690x306.png, https://aws1.discourse-cdn.com/github/optimized/2X/b/b54d7988a8771b1003c5bd562fde38ac64d9a130_2_1035x459.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/b/b54d7988a8771b1003c5bd562fde38ac64d9a130_2_1380x612.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/b54d7988a8771b1003c5bd562fde38ac64d9a130_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">7q-3qBwkteA</span><span class="informations">1920×853 373 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> I want to make a button “Try it for free”, on YouTube one person suggests using a modification of “nav__link–btn” (two -) in the “nav__link” class in html and “&amp;–btn {}” (two -) in css, I use it, but the demo does not work out what I wanted … (5 and 4 in browser)
<p>Why doesn’t the --btn modifier work for me? What should I do in this case?</p>
<pre><code class="lang-auto">=======================HTML=======================
            &lt;nav class="nav"&gt;
                &lt;a class="nav__link" href="#"&gt; Home &lt;/a&gt;
                &lt;a class="nav__link" href="#"&gt; Key Features&lt;/a&gt;
                &lt;a class="nav__link" href="#"&gt; Pricing &lt;/a&gt;
                &lt;a class="nav__link" href="#"&gt; Testiminial &lt;/a&gt;
                &lt;a class="nav__link" href="#"&gt; FAQ &lt;/a&gt;
                &lt;a class="nav__link nav__link--btn" href="#"&gt; Try for free &lt;/a&gt;
            &lt;/nav&gt;
=======================================================
</code></pre>
<pre><code class="lang-auto">========================CSS===================
.nav {
    display: flex;
    font-size:16px;
    font-weight: 700;
    text-transform: uppercase;
    font-family: 'Poppins', sans-serif;
    align-items: center

}

.nav__link {
    margin-left:50px;
    color: #242f51;
    text-decoration: none;
    opacity: 0.7;
    transition: opacity .1s linear;

    &amp;--btn {
    padding:20px 20px ;
    border:2px solid #242f51;
    border-radius: 20px;
    }
}
=====================================================
</code></pre>

**Comment by thinkverse** [comment, marked as answer]
<p>Hi there. 👋</p>
<p>Indenting your CSS that way isn’t supported by regular CSS, for that, you’d need a pre-processor like SASS which has support for those features.</p>
<pre><code class="lang-auto">.nav__link {
    margin-left:50px;
    color: #242f51;
    text-decoration: none;
    opacity: 0.7;
    transition: opacity .1s linear;
}

.nav__link--btn {
    padding:20px 20px ;
    border:2px solid #242f51;
    border-radius: 20px;
}
</code></pre>
<p>This should work just fine. :slightly_smiling_face:</p>

**Comment by baranauxyz** [comment]
<p>thank you very much!) Thanks to your advice, it worked for me)</p>
